unlock-app/src/components/interface/checkout/main/Confirm/ConfirmCard.tsx

```diff
@@ -132,6 +132,7 @@ export function ConfirmCard({
     password,
     promo,
     metadata,
+    data,
```

Comment:
I think we can probably get rid of `password`, `promo`, `captcha`... and all pack into `data` for clarity. I will work on this in a follow up PR!